lib/hsa/hsaBeamformUpchanHFB.cpp

```diff
@@ -16,8 +16,11 @@ REGISTER_HSA_COMMAND(hsaBeamformUpchanHFB);
 hsaBeamformUpchanHFB::hsaBeamformUpchanHFB(Config& config, const std::string& unique_name,
                                            bufferContainer& host_buffers,
                                            hsaDeviceInterface& device) :
-    hsaCommand(config, unique_name, host_buffers, device, "upchannelize" KERNEL_EXT,
-               "upchannelize_flip_hfb.hsaco") {
+    hsaCommand(config, unique_name, host_buffers, device,
+//               "upchannelize" KERNEL_EXT,
+//               "upchannelize_flip_hfb.hsaco") {
```

Comment:
Let's not leave old configurations in comments, this is better tracked by `git`.